terraso_backend/apps/auth/constants.py

```diff
@@ -0,0 +1 @@
+OAUTH_COOKIE_MAX_AGE_SECONDS = 300  # 5 minutes
```

Comment:
Question: What will be affeted by this expiration value?

Reply:
The parameter`max_age=OAUTH_COOKIE_MAX_AGE_SECONDS` in `middleware.py`.